Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Controller.mo

```diff
@@ -0,0 +1,2275 @@
+    "Current chiller load, in amperage"
+    annotation(Placement(transformation(extent={{-840,120},{-800,160}}),
+      iconTransformation(extent={{-140,-30},{-100,10}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput uChiAva[nChi]
+    "Chiller availability status vector"
+    annotation(Placement(transformation(extent={{-840,60},{-800,100}}),
+        iconTransformation(extent={{-140,-50},{-100,-10}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput uChiHeaCon[nChi]
+    "Chillers head pressure control status"
+    annotation (Placement(transformation(extent={{-840,-100},{-800,-60}}),
+      iconTransformation(extent={{-140,-70},{-100,-30}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput uChiWatIsoVal[nChi](
+    final min=fill(0, nChi),
+    final max=fill(1, nChi),
+    final unit=fill("1", nChi))
+    "Chilled water isolation valve position"
+    annotation(Placement(transformation(extent={{-840,-248},{-800,-208}}),
+      iconTransformation(extent={{-140,-90},{-100,-50}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.IntegerInput TChiWatSupResReq
+    "Chilled water supply temperature setpoint reset request"
+    annotation (Placement(transformation(extent={{-840,-320},{-800,-280}}),
+      iconTransformation(extent={{-140,-110},{-100,-70}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.IntegerInput chiPlaReq
+    "Number of chiller plant cooling requests"
+    annotation (Placement(transformation(extent={{-840,-360},{-800,-320}}),
+      iconTransformation(extent={{-140,-130},{-100,-90}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput uConWatPumSpe[nConWatPum](
+    final min=fill(0, nConWatPum),
+    final max=fill(1, nConWatPum),
+    final unit=fill("1", nConWatPum)) "Current condenser water pump speed"
+    annotation(Placement(transformation(extent={{-840,-410},{-800,-370}}),
+        iconTransformation(extent={{-140,-150},{-100,-110}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput uConWatPum[nConWatPum]
+    "Condenser water pump status"
+    annotation (Placement(transformation(extent={{-840,-440},{-800,-400}}),
+      iconTransformation(extent={{-140,-170},{-100,-130}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput TOut(
+    final unit="K",
+    displayUnit="degC",
+    final quantity="ThermodynamicTemperature")
+    "Outdoor air dry bulb temperature"
+    annotation (Placement(transformation(extent={{-840,-500},{-800,-460}}),
+      iconTransformation(extent={{-140,-190},{-100,-150}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput uChiCooLoa[nChi](
+    final quantity=fill("HeatFlowRate",nChi),
+    final unit=fill("W", nChi)) if have_WSE
+    "Current chiller cooling load"
+    annotation (Placement(transformation(extent={{-840,-560},{-800,-520}}),
+      iconTransformation(extent={{-140,-210},{-100,-170}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput uFanSpe(
+    final quantity="1",
+    final min=0,
+    final max=1) "Tower fan speed"
```

Comment:
- This is not an input point per §4. LIST OF POINTS.
- This is used for the fan speed control in `Buildings.Controls.OBC.ASHRAE.PrimarySystem.ChillerPlant.Towers.Controller` to cycle off fans that have been at minimum speed for too long. This should be the commanded speed `ySpeSet` computed internally.
- Lacking consistency in variable names between: `uFanSpe`, `ySpeSet`, `uMaxTowSpeSet`